src/styles/globals.css

```diff
@@ -6,13 +83,20 @@
   box-sizing: border-box;
 }
 
-body {
-  width: 100%;
-  min-height: 100vh;
-  font-size: 16px;
-  padding: 0 2em;
-}
 
+body {
+  color: rgb(var(--foreground-rgb));
+  background: linear-gradient(
+      to bottom,
+      transparent,
+      rgb(var(--background-end-rgb))
+    )
+    rgb(var(--background-start-rgb));
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+    width: 100%;
+    min-height: 100vh;
+    
```

Comment:
You are missing a closing brace for `body`

Reply:
I am going to add it, thank you